http/analysis-service/server.py

```diff
@@ -914,13 +916,18 @@ def monitor(instance_id):
     if instance.tags['Owner'] != current_user.email:
         return "No such instance: {}".format(instance_id), 404
 
+    # Instance gets killed by terminate-expired-instances.py 1 day after the creation time
+    terminate_time = parse_date(instance.launch_time, ignoretz = True) + timedelta(days = 1)
```

Comment:
Please add a `get_termination_time(begin_time)` function, used here and on line 1077.